### PR TITLE
Samples debacle

### PR DESCRIPTION
**Pull request by ngood13**
Kelsey!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! ;-)

### CODE REVIEW

**Comment by kbilsback** [comment]
Hey! @nickgood YOU are the one who wanted me to use forcats! #70

File: r_markdown/load_save_data.Rmd [hunk 1]
```diff
@@ -179,91 +179,79 @@ Reference for test id, stove and fuel types.
...
-                                                      "Built-in Justa (Envirofit, HM5000)"),
-                                        gasifiers = c("Gasifier (Envirofit, Prototype)", 
-                                                       "Gasifier (Philips, HD4012)",
-                                                       "Gasifer (Mimi Moto, Mimi Moto)"),
-                                        advanced = c("Pressurized LPG (WokSmith, WS-C1-25K)",
-                                                     "Wick Kerosene (Cook 'n Lite, 82)", 
-                                                     "Pressurized Kerosene (Butterfly, 2412)"))) %>%
-             dplyr::mutate(stovecat = forcats::lvls_reorder(stovecat, c(1, 2, 3, 4, 5)))  # set factor level order for plots
+                           fuelcat = factor(fuelcat,
+                                            levels(fuelcat)[c(3,2,4,5,1)])) %>%
```
Comment (kbilsback) [inline r_markdown/load_save_data.Rmd:238]: Why did you change my factor order back! I had it in a really specific order 😿

**Comment by nickgood** [comment]
Don’t panic! We can redo everything. Can we meet to go through the additions and make sure everything is working as expected? :-):-):-):-):-):-):-):-):-)
